MIDIchlorians/MIDIchlorians/AnimationDesignerController.swift

```diff
@@ -154,3 +154,10 @@ class AnimationDesignerController: UIViewController {
         delegate?.saveAnimation()
     }
 }
+
+extension AnimationDesignerController: PadDelegate {
+    func pad(animationUpdated animation: AnimationSequence) {
+        print(animation.animationBitsArray)
+        timelineView.frames = animation.animationBitsArray.map { ($0?.count ?? 0) > 0 }
```

Comment:
Would you prefer if I were to move this to a function in AnimationSequence, so you need not know the underlying data structure?
We can merge this in for now, and I can make another PR for the same..